src/Message/AbstractRequest.php

```diff
@@ -209,6 +210,29 @@ public function setHoldInEscrow($value)
         return $this->setParameter('holdInEscrow', (bool) $value);
     }
 
+    public function getServiceFeeAmount()
+    {

```

Comment:
@kayladnls I think so, but in the Common library they handled it this way, so I went with the established way it is handled.  https://github.com/thephpleague/omnipay-common/blob/master/src/Omnipay/Common/Message/AbstractRequest.php#L309

I'm happy to change it, if you want.